doc/datasets/index.rst

```diff
@@ -9,49 +9,60 @@ Dataset loading utilities
 The ``sklearn.datasets`` package embeds some small toy datasets
 as introduced in the :ref:`Getting Started <loading_example_dataset>` section.
 
+This package also features helpers to fetch larger datasets commonly
+used by the machine learning community to benchmark algorithms on data
+that comes from the 'real world'.
+
 To evaluate the impact of the scale of the dataset (``n_samples`` and
 ``n_features``) while controlling the statistical properties of the data
 (typically the correlation and informativeness of the features), it is
 also possible to generate synthetic data.
 
-This package also features helpers to fetch larger datasets commonly
-used by the machine learning community to benchmark algorithm on data
-that comes from the 'real world'.
-
 General dataset API
 ===================
 
-There are three distinct kinds of dataset interfaces for different types
-of datasets.
-The simplest one is the interface for sample images, which is described
-below in the :ref:`sample_images` section.
+There are three main kinds of dataset interfaces that can be used to get 
+datasets depending on the desired type of dataset.
+  
+**The dataset loaders.** They can be used to load small standard datasets, 
+described in the :ref:`toy_datasets` section.  
+
+**The dataset fetchers.** They can be used to download and load larger datasets,
+described in the :ref:`real_world_datasets` section.
 
-The dataset generation functions and the svmlight loader share a simplistic
-interface, returning a tuple ``(X, y)`` consisting of a ``n_samples`` *
+Both loaders and fetchers functions return a dictionary-like object holding 
+at least two items: an array of shape ``n_samples`` * ``n_features`` with 
+key ``data``(except for 20newsgroups) and a numpy array of 
```

Comment:
not sure why but there's formatting issue here. See https://25798-843222-gh.circle-artifacts.com/0/doc/datasets/index.html. I guess we should have a blank before the left bracket

Reply:
I didn't notice, thanks !